advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/__init__.py

```diff
@@ -1,3 +0,0 @@
-
-
-
```

Comment:
Yes they were deleted (if you look at the folder name, they were in "notUsed", so I thought now was a good time for cleanup).